File: 1/1.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(len(nums) - 1 - i):
                if nums[i] + nums[i + 1 + j] == target:
                    return [i, i + 1 + j]

print(Solution().twoSum([2, 7, 11, 15], 9)) # [0, 1]
print(Solution().twoSum([3, 2, 4], 6)) # [1, 2]
print(Solution().twoSum([-1,-2,-3,-4,-5], -8)) # [2, 4]

File: 1/1.rs
impl Solution {
  pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
      for z in 0..nums.len() {
          for y in z+1..nums.len() {
              if nums[z] == target - nums[y] {
                  return vec![z as i32, y as i32];
              }
          }
      }
      return vec![0, 0];
  }
}
